headers/functions/arm9.h

```diff
@@ -91,6 +91,17 @@ void SetBothScreensWindowsColor(int palette_idx);
 void InitMainTeamAfterQuiz(void);
 void ScriptSpecialProcess0x3(void);
 void ScriptSpecialProcess0x4(void);
+
+void InitStreamKaomado(void);
+void InitKaomado(struct kaomado*);
+void InitKaomadoSpecies(struct kaomado*, int16_t species_id);
```

Comment:
Should `species_id` be of type `enum monster_id`? The width shouldn't really matter here since it's a function param.